README.md

```diff
@@ -28,17 +28,38 @@ See the Kanban [board](https://github.com/bob-cd/bob/projects/1) to see the road
 ## Configuration
 Configuration uses the [environ library](https://github.com/weavejester/environ) and therefore several variables can be
 set by specifying them as environment variable or as java system property. Possible variables are:
-| java system properties | environment variables | defaults                    |
-|------------------------|-----------------------|-----------------------------|
-| bob-db-host            | BOB_DB_HOST           | localhost                   |
-| bob-db-port            | BOB_DB_PORT           | 5432                        |
-| bob-db-user            | BOB_DB_USER           | bob                         |
-| bob-db-name            | BOB_DB_NAME           | bob                         |
-| bob-docker-uri         | BOB_DOCKER_URI        | unix:///var/run/docker.sock |
-| bob-connect-timeout    | BOB_CONNECT_TIMEOUT   | 1000ms                      |
-| bob-read-timeout       | BOB_READ_TIMEOUT      | 30000ms                     |
-| bob-write-timeout      | BOB_WRITE_TIMEOUT     | 30000ms                     |
-| bob-call-timeout       | BOB_CALL_TIMEOUT      | 40000ms                     |
+| java system properties     | environment variables      | defaults                    |
+|----------------------------|----------------------------|-----------------------------|
+| bob-postgres-host          | BOB_POSTGRES_HOST          | localhost                   |
+| bob-postgres-port          | BOB_POSTGRES_PORT          | 5432                        |
+| bob-postgres-user          | BOB_POSTGRES_USER          | bob                         |
+| bob-postgres-name          | BOB_POSTGRES_NAME          | bob                         |
+| bob-docker-uri             | BOB_DOCKER_URI             | unix:///var/run/docker.sock |
+| bob-docker-connect-timeout | BOB_DOCKER_CONNECT_TIMEOUT | 1000ms                      |
+| bob-docker-read-timeout    | BOB_DOCKER_READ_TIMEOUT    | 30000ms                     |
+| bob-docker-write-timeout   | BOB_DOCKER_WRITE_TIMEOUT   | 30000ms                     |
+| bob-docker-call-timeout    | BOB_DOCKER_CALL_TIMEOUT    | 40000ms                     |
+
+You can also set configuration in a file called `.bob.conf` inside Bob's root folder. This is an edn-file and will be merged
+with the config passed as env-vars, system properties and the defaults. A sample configuration looks like this:
+```
+{:docker
+ {:uri "unix:///var/run/docker.sock",
+  :timeouts {:connect-timeout 1000,
+             :read-timeout 30000,
+             :write-timeout 30000,
+             :call-timeout 40000}},
+ :postgres {:host "localhost",
+            :port 5432,
+            :user "bob",
+            :database "bob"}}
+```
+
+The priority of your configuration is following:
```

Comment:
lets use the
```
1. item 1
1. item 2
...
```
scheme for ordered markdown lists.
Just to confirm the order in priority is: env, conf file, java props, defaults?

Reply:
correct